chartfx-math/src/main/java/de/gsi/math/MathBaseFast.java

```diff
@@ -76,10 +76,10 @@ private static void init() {
         sinDegLockup = new float[modulusDeg]; // sine lookup table [deg]
 
         for (int i = 0; i < modulusRad; i++) {
-            sinRadLookup[i] = (float) Math.sin((double) i / precision);
+            sinRadLookup[i] = (float) java.lang.Math.sin((double) i / precision);
         }
         for (int i = 0; i < modulusDeg; i++) {
-            sinDegLockup[i] = (float) Math.sin(Math.toRadians((double) i / precision));
+            sinDegLockup[i] = (float) java.lang.Math.sin(java.lang.Math.toRadians((double) i / precision));
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.sin' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)